src/main/java/org/jahia/modules/sam/healthcheck/probes/SupportedStackJSModulesProbe.java

```diff
@@ -25,18 +25,21 @@ public ProbeStatus getStatus() {
 
         String vmVendor = System.getProperty("java.vm.vendor", "Unknown");
         Version jvmVersion = new Version(System.getProperty("java.version", "Unknown"));
+        Version graalVMVersion = new Version(System.getProperty("org.graalvm.version", "Unknown"));
 
         // This probe is only relevant for Jahia 8.2.0.0+ in which npm-modules-engine is available. 
         // Not testing Jahia version since it is not to be backported to older versions of SAM.
         ProbeStatus status = new ProbeStatus("No issues to report", ProbeStatus.Health.GREEN);
         if (!isNpmModulesEngineStarted) {
             status = ProbeStatusUtils.aggregateStatus(status, "The environment is not running JS modules (npm-modules-engine stopped or not present)", ProbeStatus.Health.GREEN);
-        } else {
-            if (!vmVendor.contains("GraalVM")) {
-                status = ProbeStatusUtils.aggregateStatus(status, String.format("GraalVM not detected on the environment (detected vendor: %s), after switching to GraalVM make sure to enable the Javascript extension", vmVendor), ProbeStatus.Health.RED);
+        } else if (System.getProperty("org.graalvm.version", "Unknown") == "Unknown") {
```

Comment:
`System.getProperty("org.graalvm.version", "Unknown")`
is already called at like 28, may be we could introduce a new variable and reuse it.
Also no need for default value if you only want to check if a sysrtem property exists, we can check for null directly.